### PR TITLE
MapLibre GL JS v3 announcement

### PR DESCRIPTION
**Pull request by birkskyum**
The code is ready and the docs are building, so it's time to get an announcement in place before we push the button. Currently the format of the post is an ordered changelog, compiling the information from all the pre-releases.

### CODE REVIEW

File: content/news/2023-05-25-maplibre-gl-js-v3/index.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+At the bottom of the page, you'll find some demos of some of the new functionality this release brings. Enjoy!
+
+## New features and improvements
+
+- Improve control performance by restricting worker count to a max of 1 except for Safari browser. ([#2354](https://github.com/maplibre/maplibre-gl-js/pull/2354))
+- Improve control initial loading performance by forcing fadeDuration to 0 till first idle event ([#2447](https://github.com/maplibre/maplibre-gl-js/pull/2447))
+- Remove "mapbox-gl-supported" package from API. If needed, please reference it directly instead of going through
+- Resize map when container element is resized. the resize related events now has different data associated with it ([#2157](https://github.com/maplibre/maplibre-gl-js/pull/2157)) . Previously the originalEvent field was the reason of this change, for example it could be a `resize` event from the browser . Now it is `ResizeObserverEntry`, see more [here](https://developer.mozilla.org/en-US/docs/web/api/resizeobserverentry)
+- toBounds method of class LngLat is replaced by a static method fromLngLat of class LngLatBounds ([#2188](https://github.com/maplibre/maplibre-gl-js/pull/2188))
```
Comment (nyurik) [inline content/news/2023-05-25-maplibre-gl-js-v3/index.md:21]:
```suggestion
- `LngLat.toBounds()` is replaced by a static method `LngLatBounds.fromLngLat()` ([#2188](https://github.com/maplibre/maplibre-gl-js/pull/2188))
```

File: content/news/2023-05-25-maplibre-gl-js-v3/index.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+- Add pseudo (CSS) fullscreen as a fallback for iphones ([#1678](https://github.com/maplibre/maplibre-gl-js/pull/1678))
+- Add `updateData` to `GeoJSONSource` which allows for partial data updates ([#1605](https://github.com/maplibre/maplibre-gl-js/pull/1605))
+- Add a RenderPool to render tiles onto textures for 3D ([#1671](https://github.com/maplibre/maplibre-gl-js/pull/1671))
+- Add map.getCameraTargetElevation() ([#1558](https://github.com/maplibre/maplibre-gl-js/pull/1558))
+- Add `freezeElevation` to `AnimationOptions` to allow smooth camera movement in 3D ([#1514](https://github.com/maplibre/maplibre-gl-js/pull/1514), [#1492](https://github.com/maplibre/maplibre-gl-js/issues/1492))
+- Add map.setStyle's transformStyle option ([#1632](https://github.com/maplibre/maplibre-gl-js/pull/1632))
+
+## Potentially breaking changes
+
+Most of these features won't demand any changes to your code, but read carefully through the list to asses if a migration is needed.
```
Comment (nyurik) [inline content/news/2023-05-25-maplibre-gl-js-v3/index.md:72]:
```suggestion
Most of these changes will not affect your code, but read carefully through the list to asses if a migration is needed.
```

File: content/news/2023-05-25-maplibre-gl-js-v3/index.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  - WebGL2 has finally reached a level of browser support that allow us, and large projects like [Unity](https://forum.unity.com/threads/removing-support-for-gles2-and-webgl1-in-2023-1a.1360090/) to embrace it fully. This gives us lots of new features, and oportunities to improve performance though paralellization. It also removes the friction that exist when using MapLibre GL JS with libraries in the ecosystem that has moved to WebGL2 already. Most WebGL1 code will still work, as WebGL2 is mostly backwards compatible.
+- [Breaking] toBounds method of class LngLat is replaced by a static method fromLngLat of class LngLatBounds ([#2188](https://github.com/maplibre/maplibre-gl-js/pull/2188))
+- [Breaking] Make geojson data source a required field to align with the docs ([#1396](https://github.com/maplibre/maplibre-gl-js/issue/1396))
+- [Breaking] Improve control initial loading performance by forcing fadeDuration to 0 till first idle event ([#2447](https://github.com/maplibre/maplibre-gl-js/pull/2447))
+- [Breaking] Remove "mapbox-gl-supported" package from API. If needed, please reference it.
```
Comment (nyurik) [inline content/news/2023-05-25-maplibre-gl-js-v3/index.md:85]: should we list changes only once?  This seems like a dup with the above (at the top)

**Review by nyurik — COMMENTED** [review]
some minor text nits

File: content/news/2023-05-25-maplibre-gl-js-v3/index.md (hunk 3 above)
Comment (birkskyum) [reply content/news/2023-05-25-maplibre-gl-js-v3/index.md:85]: Thanks @nyurik . There could be duplicates, and we don't want them. All of this is quite directly out of the CHANGELOG.md, so I'll make sure to file a PR to the repo with the changes you propose here (both wording and duplicates).

**Comment by HarelM** [comment]
I would recommend creating a changelog of 3.0 in maplibre-gl changelog file and reference it here, not sure copying the changelog here is a good idea, maybe highlight the interesting features or some notes about improving performance and terrain bug fixes, IDK.

**Comment by birkskyum** [comment]
@HarelM , I tried to adjust it to follow your recommendation - how do you like it now?

**Comment by wipfli** [comment]
@duje can you prepare a title image for this announcement?

**Review by ovivoda — APPROVED** [review]
Great!

File: content/news/2023-05-25-maplibre-gl-js-v3/index.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ Most users will be able to update to v3 without code changes, but it's recommend
...
+[@prozessor13](https://github.com/prozessor13), [@rotu](https://github.com/rotu), [@acalcutt](https://github.com/acalcutt), [@ambientlight](https://github.com/ambientlight), [@Pessimistress](https://github.com/Pessimistress), [@HarelM](https://github.com/HarelM), [@birkskyum](https://github.com/birkskyum), [@zhangyiatmicrosoft](https://github.com/zhangyiatmicrosoft), [@pramilk](https://github.com/pramilk), [@wipfli](https://github.com/wipfli), [@xabbu42](https://github.com/xabbu42), [@ZeLonewolf](https://github.com/ZeLonewolf), [@kevinschaul](https://github.com/kevinschaul), [@zstadler](https://github.com/zstadler), [@smellyshovel](https://github.com/smellyshovel), [@jleedev](https://github.com/jleedev), [@kircher1](https://github.com/kircher1), [@llambanna](https://github.com/llambanna), [@IvanSanchez](https://github.com/IvanSanchez), [@tordans](https://github.com/tordans), [@ChrisLoer](https://github.com/ChrisLoer), [@cns](https://github.com/cns), [@vanilla](https://github.com/vanilla), [@lukashass](https://github.com/lukashass), [@VehpuS](https://github.com/VehpuS),
```
Comment (wipfli) [inline content/news/2023-05-25-maplibre-gl-js-v3/index.md:16]: Should be @cns-solutions-admin maybe?

Comment (birkskyum) [reply content/news/2023-05-25-maplibre-gl-js-v3/index.md:16]: Oh yes, the dash tripped me - same with vanilla --> vanilla-lake I see

**Comment by duje** [comment]
gl-js-v3

File: content/news/2023-05-25-maplibre-gl-js-v3/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
+---
```
Comment (duje) [inline content/news/2023-05-25-maplibre-gl-js-v3/index.md:1]: See Open Graph and Twitter Cards: https://gohugo.io/templates/internal/.

I would suggest to add "description" and "images" in Front Matter for this post. "images" is missing under "params": https://github.com/maplibre/maplibre.github.io/blob/main/config.toml#L24.

Comment (duje) [reply content/news/2023-05-25-maplibre-gl-js-v3/index.md:1]: i.e. images: ['gl-js-v3.png'] if https://github.com/maplibre/maplibre.github.io/pull/171#issuecomment-1559212865 is to be used as a preview

Comment (duje) [reply content/news/2023-05-25-maplibre-gl-js-v3/index.md:1]: As in https://github.com/maplibre/maplibre.github.io/blob/main/layouts/partials/head.html#L23 add 

images = "/img/share-image.png"

to https://github.com/maplibre/maplibre.github.io/blob/main/config.toml#L24

Comment (duje) [reply content/news/2023-05-25-maplibre-gl-js-v3/index.md:1]: share-image.png currently is https://github.com/maplibre/maplibre.github.io/blob/main/static/img/share-image.png

**Comment by duje** [comment]
gl-js-v3

**Review by duje — COMMENTED** [review]
:clap:

**Pull request merged**